atomic_client.py

```diff
@@ -77,6 +77,10 @@ def images(self):
     def vulnerable(self):
         return self.dbus_object.VulnerableInfo(dbus_interface="org.atomic", timeout = 2147400)
 
+    @polkit.enable_proxy
+    def ps(self):
+        return self.dbus_object.Ps(dbus_interface="org.atomic", timeout = 2147400)
```

Comment:
If the timeout limit is not set, then there is the possibility that there can be a timeout before the data is sent back and output.